data/crud/fund_round_queries.py

```diff
@@ -5,6 +5,10 @@
 from pre_award.db import db
 
 
+def get_fund(fund_short_name: str) -> Fund | None:
+    return db.session.scalar(select(Fund).where(Fund.short_name == fund_short_name.upper()))
```

Comment:
I feel like we should be using a CITEXT column, maybe - easier than having to remember to uppercase this thing everywhere in code? 🤔